lock/src/main/java/org/infinispan/lock/impl/manager/EmbeddedClusteredLockManager.java

```diff
@@ -96,14 +118,25 @@ public ClusteredLockConfiguration getConfiguration(String name) {
 
    @Override
    public boolean isDefined(String name) {
+      if (trace) {
+         log.tracef("LOCK[%s] isDefined has been called", name);
+      }
       if (cache == null) {
          cache = extractCacheHolder(cacheHolderFuture).getClusteredLockCache();
       }
       return cache.containsKey(new ClusteredLockKey(ByteString.fromString(name)));
    }
 
+   @ManagedOperation(
```

Comment:
you can't reuse this methods. 
`CompletableFuture` isn't serializable and the operation will fail (in JConsole at least) with `NotSerializableException`

Reply:
ok, I need to make them sync ?